### PR TITLE
Add documentation for installation on older BIOS's which don't detect…

### LINKED ISSUE #1760: [Generic x86-64] Same physical SSD imaged with 7.4 can boot if connected via external USB but not from internal SATA
### Describe the issue you are experiencing

I verified with multiple experiments that if I image an SSD with HA OS 7.4 (also tried 8.0-dev and 6.6) and connect it via the SATA port, the disk seems not imaged in a way that the BIOS recognizes its UEFI partition. So, it doesn't boot: in the best case I see a black screen.

It can boot if I disconnect the SSD from the SATA slot and connect it to the external USB3 port: in that case, the BIOS sees its UEFI partition correctly.

After a long discussion (and support: thanks folks!) in the #installation Discord channel, I decided to file an issue.

### What operating system image do you use?

generic-x86-64 (Generic UEFI capable x86-64 systems)

### What version of Home Assistant Operating System is installed?

7.4, 8.0-dev, 6.6

### Did you upgrade the Operating System.

No

### Steps to reproduce the issue

1. Point BalenaEtcher to https://github.com/home-assistant/operating-system/releases/download/7.4/haos_generic-x86-64-7.4.img.xz (tried with 8.0-dev and 6.6 as well)
2. Burn the image onto a SATA SSD
3. Try to boot from SATA SSD on a Fujitsu Esprimo q920 (i5-4590t) with default BIOS settings (no Secure Boot, UEFI boot, etc., was able to boot a standard Debian 11)
4. Try to boot while the *same* SATA SSD drive is connected via the external USB3 port (via a SATA-to-USB connector).

### Anything in the Supervisor logs that might be useful for us?

```txt
Black screen
```

### Anything in the Host logs that might be useful for us?

```txt
Black screen
```

### System Health information

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tsvi**
… the EFI entry correctly

Fixes home-assistant/operating-system#1760

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -157,6 +157,23 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
 3. Power the system on.
 
    - Wait for the Home Assistant welcome banner to show up in the console of the generic-x86-64 system.
+
+<div class="note">
+If the machine complains about not being able to find a bootable medium, you might need to specify the EFI entry in your BIOS.
+This can be accomplished either by using a live operating system (e.g. Ubuntu) and running the following command:
+
+  ```text
+  efibootmgr --create --disk /dev/sda --part 1 --label "HAOS" --loader \EFI\BOOT\bootx64.efi
```
Comment (agners) [inline source/_includes/installation/operating_system.md]: I'd prefer adding a newline here so it looks nicer in the docs, e.g.

```
efibootmgr --create --disk /dev/sda --part 1 --label "HAOS" \
    --loader \EFI\BOOT\bootx64.efi
```

Also, can you replace `sda` with a placeholder and note that it should point to the drive where Home Assistant OS has been installed to, typically `sda` or `nvme0n1` if a NVMe SSD is used.

**Review by agners — CHANGES_REQUESTED** [review]
It makes sense to add this as some systems indeed require it to be able to boot HAOS :+1:

**Review by agners — DISMISSED** [review]

File: source/_includes/installation/operating_system.md [hunk 2]
```diff
@@ -157,6 +157,24 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
 3. Power the system on.
 
    - Wait for the Home Assistant welcome banner to show up in the console of the generic-x86-64 system.
+
+<div class="note">
```
Comment (frenck) [inline source/_includes/installation/operating_system.md:161]:
```suggestion
<div class="note">

```

**Review by frenck — APPROVED** [review]
Thanks, @tsvi 👍

**Pull request merged**